rpcd/playbooks/roles/rpc_maas/tasks/package_install.yml

```diff
@@ -21,9 +21,23 @@
     cache_valid_time: 600
   with_items: maas_apt_packages
 
-- name: Install pip packages
+#- name: Install pip packages
+#  pip:
+#   name: "{{ item }}"
+#   state: present
+#   extra_args: "{{ pip_install_options | default('') }}"
+#  with_items: maas_pip_packages + maas_pip_dependencies
+
+-name: Install pip packages to venv
   pip:
     name: "{{ item }}"
     state: present
     extra_args: "{{ pip_install_options | default('') }}"
+    virtualenv: "{{ maas_venv_bin | dirname }}"
```

Comment:
You might want to have this instead:

virtualenv: "{{ maas_venv_enabled | bool | ternary(maas_venv_bin|dirname, omit) }}"
virtualenv_site_packages: "{{ maas_venv_enabled | bool | ternary('no', omit) }}"